saleor/graphql/product/mutations/attributes.py

```diff
@@ -45,6 +49,43 @@ class Meta:
     def user_is_allowed(cls, user, input):
         return user.has_perm('product.manage_products')
 
+    @classmethod
+    def clean_input(cls, info, instance, input, errors):
+        cleaned_input = super().clean_input(info, instance, input, errors)
+
+        values = cleaned_input.get('values', [])
+        names = [value['name'] for value in values]
+        if len(set(names)) != len(names):
+            cls.add_error(
+                errors, 'values', 'Duplicated attribute value names provided.')
+        for value_data in values:
+            attribute_value = models.AttributeValue(**value_data)
+            try:
+                attribute_value.full_clean()
+            except ValidationError as validation_errors:
+                for field in validation_errors.message_dict:
+                    # Attribute instance is not created yet so we cannot add it
+                    if field == 'attribute':
+                        continue
+                    for message in validation_errors.message_dict[field]:
+                        error_field = 'values-%(field)s' % {'field': field}
```

Comment:
We implemented a similar idea [in other mutation](https://github.com/mirumee/saleor/blob/master/saleor/graphql/product/mutations/products.py#L347) but used colons to separate field and attribute name. I'd use the same format here. Later we should think of some standardization and probably a helper method to generate such errors.